tiatoolbox/utils/exceptions.py

```diff
@@ -27,3 +27,11 @@ class FileNotSupported(Exception):
     def __init__(self, message="File format is not supported"):
         self.message = message
         super().__init__(self.message)
+
+
+class MethodNotSupported(Exception):
```

Comment:
Can you write a quick test for this code for coverage?

Reply:
Added in[ this commit](https://github.com/TIA-Lab/tiatoolbox/pull/35/commits/a5d3d7f6097a144f92d1e5b1cc183ed1464bfbfd).